ckan/templates/snippets/search_form.html

```diff
@@ -68,7 +68,7 @@ <h1>
               {%- else -%}
                 {{ h.list_dict_filter(search_facets_items, 'name', 'display_name', value) }}
               {%- endif %}
-              <a href="{{ facets.remove_field(field, value) }}" class="remove" title="{{ _('Remove') }}"><i class="fa fa-times"></i></a>
+              <a href="{{ facets.remove_field(field, value) }}" class="remove" aria-label="{{ _('Remove applied filter: {value}').format(value=value) }}" data-bs-title="{{ _('Remove applied filter: {value}').format(value=value) }}" data-bs-toggle="tooltip"><i class="fa fa-times"></i></a>
```

Comment:
I recommend leaving only `Remove` in inside data-bs-title. Aria-label is good, but tooltip for user that can see the item is too verbose